### ISSUE TITLE
Steps missing for "Managed Google Play private (LOB) app publishing directly in the Intune console" process

### ISSUE THREAD

**Issue by rasldasl** [MicrosoftDocs/memdocs#1196]
[Enter feedback here]
After using this process to create the app there are further steps needed to approve the app so it will show up in Intune. 
I opened a case and was given these instructions. Approval after step 9 before syncing. You can sync all day long but the newly created app cannot be approved from the iframe (app creation page). 

•	Go to Tenant Administration > Connectors and tokens > Managed Google Play
Follow the numbered steps here (details added for step 1).
1. Go to the managed Google Play store to approve applications for your enterprise
Open the managed Google Play store https://play.google.com/work and go to My Managed apps (direct link https://play.google.com/work/licenses/apps)
Click on the newly created Private app (padlock icon) and approve).
2. Sync.
 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a5a2b8f0-f2a0-f4d6-94d9-06e6d9632f59
* Version Independent ID: 8c8d89cf-9ed7-bb02-2439-e7ce9bb03f0c
* Content: [Add and assign Managed Google Play apps to Android Enterprise devices - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/apps/apps-add-android-for-work#managed-google-play-private-lob-app-publishing-directly-in-the-intune-console)
* Content Source: [memdocs/intune/apps/apps-add-android-for-work.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/apps/apps-add-android-for-work.md)
* Service: **microsoft-intune**
* Sub-service: **apps**
* Technology: ****
* GitHub Login: @Erikre
* Microsoft Alias: **erikre**

**Comment by Erikre** [comment]
Thanks for the feedback. I'm making updates to the topic based on your suggestion now.

**Comment by Erikre** [comment]
In addition to the above update, I'm looking into this issue a bit future to see if I can add additional info. I've created an internal work item to track this potential doc addition.

**Comment by rasldasl** [comment]
I think some part of the workflow was broken when I reported the issue. The next time I added an app it got approved automatically. Still, it’s helpful to know how to check and approve if necessary.

Sent from my iPad

On Feb 18, 2021, at 5:04 PM, Erik Reitan <notifications@github.com> wrote:

﻿

In addition to the above update, I'm looking into this issue a bit future to see if I can add additional info. I've created an internal work item to track this potential doc addition.
